cypress/e2e/marc/marc-bibliographic/edit-marc-bib/unlink-created-unsaved-linked-marc-bib-field-from-marc-authority-record.cy.js

```diff
@@ -0,0 +1,139 @@
+            Logs.openFileDetails(marcFile.fileName);
+            Logs.getCreatedItemsID().then((link) => {
+              createdRecordIDs.push(link.split('/')[5]);
+            });
+          },
+        );
+      });
+      cy.login(testData.userProperties.username, testData.userProperties.password, {
+        path: TopMenu.inventoryPath,
+        waiter: InventoryInstances.waitContentLoading,
+      }).then(() => {
+        InventorySearchAndFilter.selectSearchOptions(testData.searchOption, testData.marcBibTitle);
+        InventorySearchAndFilter.clickSearch();
+        InventoryInstances.selectInstanceById(createdRecordIDs[2]);
+        InventoryInstance.waitLoading();
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken(() => {
+      Users.deleteViaApi(testData.userProperties.userId);
+    });
+    MarcAuthority.deleteViaAPI(createdRecordIDs[0]);
+    MarcAuthority.deleteViaAPI(createdRecordIDs[1]);
+    InventoryInstance.deleteInstanceViaApi(createdRecordIDs[2]);
+  });
+
+  it(
+    'C366554 Unlink created unsaved linked "MARC Bib" field from "MARC Authority" record (spitfire) (TaaS)',
+    { tags: ['extendedPath', 'spitfire'] },
+    () => {
+      InventoryInstance.editMarcBibliographicRecord();
+      QuickMarcEditor.addNewField('700', '', 12);
+      QuickMarcEditor.addNewField('700', '$a Sabino, J.', 13);
+      QuickMarcEditor.clickLinkIconInTagField(13);
+      MarcAuthorities.switchToSearch();
+      MarcAuthorities.searchBy('Keyword', 'Sprouse, Chris');
+      InventoryInstance.selectRecord();
+      MarcAuthorities.clickLinkButton();
+      QuickMarcEditor.verifyAfterLinkingAuthorityByIndex(13, 700);
+      QuickMarcEditor.verifyTagFieldAfterLinking(
+        13,
+        '700',
+        '\\',
+        '\\',
+        '$a Sprouse, Chris',
+        '',
+        '$0 id.loc.gov/authorities/names/nb98017694',
+        '',
+      );
+      QuickMarcEditor.clickLinkIconInTagField(14);
+      MarcAuthorities.switchToSearch();
+      MarcAuthorities.searchBy('Keyword', 'Sabino, Joe');
+      InventoryInstance.selectRecord();
+      MarcAuthorities.clickLinkButton();
+      QuickMarcEditor.verifyAfterLinkingAuthorityByIndex(14, 700);
+      QuickMarcEditor.verifyTagFieldAfterLinking(
+        14,
+        '700',
+        '\\',
+        '\\',
+        '$a Sabino, Joe',
+        '',
+        '$0 id.loc.gov/authorities/names/no2011137752',
+        '',
+      );
+      QuickMarcEditor.clickUnlinkIconInTagField(13);
+      QuickMarcEditor.confirmUnlinkingField();
+      QuickMarcEditor.verifyTagFieldAfterUnlinking(13, '700', '\\', '\\', '');
+      QuickMarcEditor.clickUnlinkIconInTagField(14);
+      cy.wait(500); // wait until confirmation panel will appear
+      QuickMarcEditor.confirmUnlinkingField();
+      QuickMarcEditor.verifyTagFieldAfterUnlinking(14, '700', '\\', '\\', '$a Sabino, J.');
+      QuickMarcEditor.deleteField(13);
+      QuickMarcEditor.deleteField(13);
```

Comment:
Check after deletion notification please, "No Undo displayed" as it says in test case, for example this checks if there is undo displayed "cy.get('[class^=deletedRowPlaceholder-]').contains('span', 'Undo');" but you have to check that it is not there.